.github/workflows/nodejs-windows.yml

```diff
@@ -0,0 +1,32 @@
+name: Docusaurus on Windows
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: windows-2019
+    strategy:
+      matrix:
+        node-version: [8.x]
```

Comment:
Since we have a minimum required Node version, is it possible to use that here?

Reply:
You have to pin exact version. Or just use .x which means the latest stable version of that major node version. Our min is 8.10.0 anyway which is the earlier version of 8.

Reply:
Well this is the same as our circleci config. 8.x